docs/commands/GetArchiveDaysMC.md

```diff
@@ -0,0 +1,136 @@
+
+It's a command with a [two-bytes header](../message.md#command-with-a-two-bytes-header).
+
+### Parameters
+
+#### **date**
+
+Start date for requested day pulse counter's values.
+<br>
+[See details](../../types.md#packed-date).
+
+#### **channels bit set**
+
+[See details](../../types.md#channels-bit-set).
+
+#### **days**
+
+Amount of days to get data from archive.
+
+### Examples
+
+#### channels `1`, `3`, `4`:
+
+| Field        | Value                     | Bits                 | Hex      |
+|--------------|---------------------------|----------------------|----------|
+| command id   | `27`                      |                      | `0x1b`   |
+| command size | `4`                       |                      | `0x04`   |
+| date         | `2023.12.23 00:00:00 GMT` | `0b0010111110010111` | `0x2f97` |
+| channels     | `1`, `3`, `4`             | `0b00001101`         | `0x0d`   |
+| days         | `2`                       |                      | `0x02`   |
+
+Message hex dump with LRC: `1b 04 2f 97 0d 02 fd`
+
+
+## Response
+
+### Format
+
+| Size         | Type                                         | Field                                 |
+| ------------ | -------------------------------------------- | ------------------------------------- |
+| `1`          | `byte`                                       | command id = `0x1b`                   |
+| `1`          | `byte`                                       | command size (dynamic, `5+`)          |
+| `2`          | [packed date](../types.md#packed-date)       | [start date](#date)                   |
+| `1..5`       | [extended value](../types.md#extended-value) | [channels bit set](#channels-bit-set) |
+| `1`          | `byte`                                       | [amount of days](#days)               |
+| `1..5`       | [extended value](../types.md#extended-value) | channel `1` day `1` value             |
+| `1..5`       | [extended value](../types.md#extended-value) | channel `1` day `2` value             |
+| `1..5`       | [extended value](../types.md#extended-value) | channel `2` day `1` value             |
+| `1..5`       | [extended value](../types.md#extended-value) | channel `2` day `2` value             |
+| ...          | ...                                          | ...                                   |
+| `1..5`       | [extended value](../types.md#extended-value) | channel `N` day `D` value             |
+| `1..5`       | [extended value](../types.md#extended-value) | channel `N` day `D` value             |
+
+It's a command with a [two-bytes header](../message.md#command-with-a-two-bytes-header).
+
+### Parameters
+
+#### **date**
+
+Start date for requested day pulse counter's values.
+<br>
+[See details](../../types.md#packed-date).
+
+#### **channels bit set**
+
+[See details](../types.md#channels-bit-set).
+
+#### **days**
+
+Amount of days from archive.
+
+#### **channel values by days**
```

Comment:
this section does not exist anymore so let's remove it

Reply:
removed